packages/schemas/src/consts/storage.ts

```diff
@@ -0,0 +1,13 @@
+export const maxUploadFileSize = 8 * 1024 * 1024; // 8MB
+
+// Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+export const allowUploadMimeTypes = [
```

Comment:
Should we put it under the schemas? It seems unrelated to the main auth flow. More like silverhand essentials or sth

Reply:
It's ok to put it under the schemas since this array is also used to construct types(DTO) in both the backend and front end.